components/user/UserMilestone.js

```diff
@@ -10,7 +10,7 @@ export default function UserMilestone({ milestone, isGoal }) {
     <li
       className={`py-4 ${isGoal ? "opacity-50" : ""}`}
       style={{
-        borderColor: milestone.color,
+        borderColor: "lightgrey",
```

Comment:
This can be achieved with a tailwind css class and the style is no longe required